neuromation/api/storage.py

```diff
@@ -338,11 +338,18 @@ async def _rm(
         ) as resp:
             if resp.headers.get("Content-Type", "").startswith("application/x-ndjson"):
                 async for line in resp.content:
-                    remove_listing = json.loads(line)
+                    server_message = json.loads(line)
+                    if "error" in server_message:
+                        err_text = server_message["error"]
+                        os_errno = server_message.get("errno", None)
+                        if os_errno is not None:
+                            os_errno = errno.__dict__.get(os_errno, os_errno)
```

Comment:
I'm missed. What is `errno` type here?

Reply:
`errno` here is `module` instance (it comes from `import errno`). This is code is almost same as in `_Core.request`.

Reply:
Got you